<launch>
  <!-- Launch Head -->
  <!-- <include file="$(find tracker)/launch/head.launch"/> -->
  <include file="$(find tracker)/launch/tf.launch"/>

  <include file="$(find bender_base)/launch/base.launch"/>

  <!-- Launch Asus Camera -->
  <group ns="bender/sensors">
    <include file="$(find openni2_launch)/launch/openni2.launch">
      <arg name="camera" value="rgbd_head" />
      <arg name="publish_tf" value="false" />
      <arg name="depth_frame_id" value="bender/sensors/rgbd_head_depth_optical_frame" />
      <arg name="rgb_frame_id" value="bender/sensors/rgbd_head_rgb_optical_frame" />
    </include> 
  </group>

  <!-- Run Tracker node -->
  <node name="tracker" pkg="tracker" type="follow_me.py" output="screen" />
</launch>